app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -17,7 +17,9 @@
  */
 package com.infomaniak.mail.ui
 
+import android.app.Activity
 import android.app.Application
+import android.content.Context
```

Comment:
Also, you are in an `AndroidViewModel`. You already have the context inside the `ViewModel`. No need to import `Context` and also `Activity`